Surface_mesher/include/CGAL/IO/Complex_2_in_triangulation_3_to_medit.h

```diff
@@ -63,8 +63,9 @@ output_surface_facets_to_medit (std::ostream& os, const C2t3& c2t3)
       vit != tr.finite_vertices_end();
       ++vit)
   {
-    if(V.find(vit) != V.end()){
-      V[vit] = inum++;
+    auto it = V.find(vit);
+    if(it != V.end()){
+      *it = inum++;
```

Comment:
Actually, my code it probably wrong:
```suggestion
      it->second = inum++;
```

You should test.